MEME/MEME/Model/ModelReservationView/ModelReservationChartView/ModelReservationChartTableViewController.swift

```diff
@@ -0,0 +1,307 @@
+        button.layer.cornerRadius = 17
+        
+        
+        return button
+    }()
+    private var reservationChartTableView: UITableView!
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+
+        setupButtonsStackView()
+        setupButtonsAction()
+        selectButton(allButton)
+        setupReservationChartTableView()
+        configureSubviews()
+        makeConstraints()
+    }
+    
+    // MARK: - configureSubviews
+    func configureSubviews() {
+        view.addSubview(backButton)
+        view.addSubview(titleLabel)
+        buttonScrollView.showsHorizontalScrollIndicator = false
+        view.addSubview(buttonScrollView)
+        view.addSubview(lineView)
+        view.addSubview(numLabel)
+        view.addSubview(searchNumLabel)
+        view.addSubview(sortButton)
+        view.addSubview(reservationChartTableView)
+        
+    }
+    
+    // MARK: - makeConstraints
+    func makeConstraints() {
+        backButton.snp.makeConstraints {make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(16)
+            make.leading.equalToSuperview().offset(24)
+            make.width.equalTo(40)
+            make.height.equalTo(40)
+        }
+        titleLabel.snp.makeConstraints {make in
+            make.centerY.equalTo(backButton.snp.centerY)
+            make.centerX.equalToSuperview()
+        }
+        buttonScrollView.snp.makeConstraints {make in
+            make.top.equalTo(titleLabel.snp.bottom).offset(14.5)
+            make.leading.equalToSuperview().offset(24)
+            make.trailing.equalToSuperview()
+            make.height.equalTo(48)
+        }
+        lineView.snp.makeConstraints {make in
+            make.top.equalTo(buttonScrollView.snp.bottom)
+            make.leading.equalToSuperview().offset(24)
+            make.trailing.equalToSuperview().offset(-24)
+            make.height.equalTo(1)
+        }
+        sortButton.snp.makeConstraints {make in
+            make.top.equalTo(lineView.snp.bottom).offset(17)
+            make.trailing.equalToSuperview().offset(-24)
+        }
+        numLabel.snp.makeConstraints {make in
+            make.centerY.equalTo(sortButton.snp.centerY)
+            make.leading.equalToSuperview().offset(24)
+        }
+        searchNumLabel.snp.makeConstraints {make in
+            make.centerY.equalTo(numLabel.snp.centerY)
+            make.leading.equalTo(numLabel.snp.trailing)
+        }
+        reservationChartTableView.snp.makeConstraints {make in
+            make.top.equalTo(sortButton.snp.bottom)
+            make.leading.equalToSuperview().offset(24)
+            make.trailing.equalToSuperview().offset(-24)
+            make.bottom.equalToSuperview()
+        }
+    }
+    //MARK: -Action
+    private func setupButtonsAction() {
+            let buttons = [allButton, specialButton, actorButton, interviewButton, dailyButton, studioButton, weddingButton, partyButton, etcButton]
```

Comment:
```suggestion
            let buttons = [
                allButton,
                specialButton,
                actorButton,
                interviewButton,
                dailyButton,
                studioButton,
                weddingButton,
                partyButton,
                etcButton
            ]
```

위와 같이 작성하면 조금 더 코드의 가독성을 향상시킬 수 있을 것 같습니다!